execution/execution-correctness/Cargo.toml

```diff
@@ -28,7 +28,7 @@ thiserror = "1.0"
 [dev-dependencies]
 config-builder = { path = "../../config/config-builder", version = "0.1.0" }
 executor-utils = { path = "../executor-utils", version = "0.1.0" }
-libradb = { path = "../../storage/libradb", version = "0.1.0"}
+libradb = { path = "../../storage/libradb", version = "0.1.0", features = ["fuzzing"] }
```

Comment:
I suspect these changes may need to go to the `[features]` section below instead. I'm not sure.

Reply:
I think so, too.

Reply:
You do that when libradb is in the main "[dependency]" section

Reply:
Actually I'm not sure the guidance is very clear in this space.

The reality is if you specify this under dev-dependencies you could accidentally enable the feature via transitive dependencies. So despite it seeming somewhat unclear.. I think the suggestions are correct.

cc: @bmwill

Reply:
"The reality is if you specify this under dev-dependencies you could accidentally enable the feature via transitive dependencies." elaboration? @davidiw

Reply:
@sunshowers is likely to make a lint at some point soon, so not sure accidental enablement will be a concern, but not there yet.  Unfortunately they are out this week as is @bmwill, so @metajack thoughts?

Reply:
This is fine because it does not also appear in [dependencies], which means it won't get merged together. dev-dependencies are not considered except for the current crate, so it won't get merged into another crate either.